### ISSUE TITLE
FetchError: request to https://www.overleaf.com/project failed, reason: certificate is not yet valid

### ISSUE THREAD

**Issue by Irislucent** [iamhyc/Overleaf-Workshop#115]
### Describe the bug
> A clear and concise description of what the bug is.

I have trouble logging in back to my Overleaf account after the last cookie expired. The extension gives the output: "FetchError: request to https://www.overleaf.com/project failed, reason: certificate is not yet valid". 

### Expected behavior
> A clear and concise description of what you expected to happen.

I should be able to log in smoothly like before, because I copied the cookie from my browser as guided. The only difference is that my cookie shown on browser doesn't start with "overleaf_session2" or "sharelatex".
![image](https://github.com/iamhyc/Overleaf-Workshop/assets/49158126/45e09614-7bc4-4c72-8076-d26c8a6482be)

### How To Reproduce
> Detailed steps to reproduce the behavior.

Logging in back using new cookie.

### Environment
 - **Overleaf Workshop Extension version**: [e.g., 0.7.0] 0.11.0
 - **VS Code version**: [e.g. 1.85.0] 1.87.0
 - **Overleaf Edition**: Official / Community / Enterprise (Server Pro) Premium by Institution

### [Optional] Developer Logs
> If applicable, please paste the logs inside the details section below.
> Please find the logs via: Title bar **"Help"** > **"Toggle Developer Tools"** > **"Console"**.

<details>
  ...
</details>

**Comment by iamhyc** [comment]
Hi, I cannot reproduce your situation. Please check your network connection following the guidance here:

https://github.com/iamhyc/Overleaf-Workshop/blob/master/docs/wiki.md#q3-why-the-extension-loses-connection-to-the-server-frequently-or-loading-indefinitely

**Comment by iamhyc** [comment]
> I should be able to log in smoothly like before, because I copied the cookie from my browser as guided. The only difference is that my cookie shown on browser doesn't start with "overleaf_session2" or "sharelatex".

There are three cookie in your screenshot, separated by the comma. Please only use the `overleaf_session2=...` one.

P.S. Please make sure the cookies in your screenshot have been invalidated or other people can login with your cookies :)

**Comment by Irislucent** [comment]
After several trials and errors, I found that a VPN software in my background (even when unactivated) prevents logging in. I could log in using cookies again after quitting the VPN. So it's not the extension's problem.

I also found that disconnecting to overleaf in vscode will reset my login status in browser too (and I get a new cookie after re-login, of course). This is very interesting and I don't understand why...

Anyways, thank you for your quick reply. This extension has been really helpful.
